tmt/steps/provision/__init__.py

```diff
@@ -97,6 +357,8 @@ class Guest(tmt.utils.Common):
     role: Optional[str]
     guest: Optional[str]
 
+    # facts: GuestFacts
```

Comment:
A comment here would be also nice, that we cannot define it like this here, and we need the magic property

Reply:
Removed in 1e603a23f05f7aa894c5ab5875acf049e8548ed0.